src/commands/core/asset/create.ts

```diff
@@ -204,11 +204,11 @@ export default class AssetCreate extends TransactionCommand<typeof AssetCreate>
     return jsonUri
   }
 
-  private formatAssetResult(result: any, explorer: ExplorerType): string {
+  private async formatAssetResult(result: any, explorer: ExplorerType): Promise<string> {
     return `--------------------------------
   Asset: ${result.asset}
   Signature: ${base58.deserialize(result.signature! as Uint8Array)[0]}
-  Explorer: ${generateExplorerUrl(explorer, base58.deserialize(result.signature! as Uint8Array)[0], 'transaction')}
+  Explorer: ${await generateExplorerUrl(explorer, this.context.rpcUrl, base58.deserialize(result.signature! as Uint8Array)[0], 'transaction')}
```

Comment:
nit: there is a util function `txSignatureToString` in the util.ts you can use for this line and the line above